draft/spec/index.html

```diff
@@ -911,10 +911,13 @@ <h2>Version Inventory and Inventory Digest</h2>
             Every OCFL Object <span id="E063">MUST</span> have an inventory file within the OCFL Object Root,
             corresponding to the state of the OCFL Object at the current version. Additionally, every version directory
             <span id="W010">SHOULD</span> include an inventory file that is an <a href="#inventory">Inventory</a> of
-            all content for versions up to and including that particular version. Where an OCFL Object contains
-            <code>inventory.json</code> in version directories, the inventory file in the OCFL Object Root
-            <span id="E064">MUST</span> be the same as the file in the most recent version. See also requirements for
-            the corresponding <a href="#inventory-digest">Inventory Digest</a>.
+            all content for versions up to and including that particular version. In the case where no files have been
```

Comment:
I think we could make this cleaner by starting a new paragraph at 

> In the case where.....

But I'm not going to hold up approval for that simple change.